### PR TITLE
Adding --name option to make and run commands

### PR DESCRIPTION
**Pull request by igal-stratoscale**
@shay-stratoscale @eran-stratoscale @ravidbro please review

### CODE REVIEW

File: skipper/cli.py [hunk 1]
```diff
@@ -154,10 +154,11 @@ def rmi(ctx, remote, image, tag):
 
 @cli.command(context_settings=dict(ignore_unknown_options=True))
 @click.option('-i', '--interactive', help='Interactive mode', is_flag=True, default=False, envvar='SKIPPER_INTERACTIVE')
+@click.option('-n', '--name', help='container name', default=None)
```
Comment (rom-stratoscale) [inline skipper/cli.py]: Capital C

File: skipper/cli.py [hunk 2]
```diff
@@ -171,18 +172,20 @@ def run(ctx, interactive, env, command):
...
                       interactive=interactive,
+                      name=name,
                       net=ctx.obj['build_container_net'],
                       volumes=ctx.obj.get('volumes'),
                       workdir=ctx.obj.get('workdir'))
 
 
 @cli.command(context_settings=dict(ignore_unknown_options=True))
 @click.option('-i', '--interactive', help='Interactive mode', is_flag=True, default=False, envvar='SKIPPER_INTERACTIVE')
+@click.option('-n', '--name', help='container name', default=None)
```
Comment (rom-stratoscale) [inline skipper/cli.py]: Capital C

File: skipper/cli.py (hunk 1 above)
Comment (rom-stratoscale) [inline skipper/cli.py]: Isnt it better to add it as a last parameter - for backward compatibility?

Comment (igal-stratoscale) [inline skipper/cli.py]: dunno.
From this point you are right but logically this place is better as for me.
@eran-stratoscale what do u think?

**Comment by ravidbro** [comment]
I'm suggesting to add new field 'extra_params' which will be a string that will be added to the docker command.
every week, someone adds another delegation to another parameter in docker cli.
I'm sure this is not the last parameter and next week, someone will need something else

File: skipper/cli.py (hunk 1 above)
Comment (eran-stratoscale) [inline skipper/cli.py]: @rom-stratoscale what do you mean by  backward compatibility?
It looks fine to me.

**Comment by igal-stratoscale** [comment]
@shay-stratoscale @eran-stratoscale  @ravidbro @rom-stratoscale  can i push it after fixes? There is some process that i to do before pushing?

**Comment by ravidbro** [comment]
1. You didn't fix @rom-stratoscale 's comment
2. you need to tag the version and publish it to the global pypi after you push to master

**Comment by igal-stratoscale** [comment]
fixed

**Comment by rom-stratoscale** [comment]
Please lets push it.
I agree with @ravidbro. The next one that will add a field, should do it. ok?

**Comment by rom-stratoscale** [comment]
:+1: 
Lets upgrade the slaves and add the cleanup logic to Jenkins.
@igal-stratoscale @roeigo-stratoscale

**Pull request merged**